instructor-training-process.md

```diff
@@ -0,0 +1,154 @@
+	Groups must also commit to organizing and running at least one workshop within a few months of training.
+    1.  We may commit to training people for partner organizations as part of their partnership agreements.
+        Here,
+	we negotiate dates with the partner site,
+	which selects the participants subject to the same per-site size restrictions above (3-12 people per site).
+1.  **Matching candidates to courses.**
+    This has been done manually to date:
+    the lead trainer, members of the Steering Committee, and/or the Executive Directors
+    decide on dates and then invite individuals and/or groups to take part.
+1.  **Running courses.**
+    1.  In-person courses are run over two days,
+        with all participants at one site.
+    1.  Live online courses are also run over two days,
+        but participants take part in small groups at a small number of sites
+	(typically 3-12 per site and 2-4 sites).
+    1.  Asynchronous online courses are run over several weeks.
+        Participants are typically *not* co-located,
+	but get together for an hour of discussion online each week.
+	This format is being phased out,
+	as it took more effort from trainers
+	and had poorer follow-through rates than the intensive two-day versions.
+1.  **Homework.**
+    1.  After completing the course,
+        the trainee picks one core lesson from each carpentry she wants to teach
+        and submits a small change for that lesson.
+    1.  Changes do *not* have to be accepted in order for the trainee to move on,
+        but they do have to be properly formatted.
+    1.  Changes are submitted as GitHub pull requests for Software Carpentry
+        and by appending them to a Google Docs page for Data Carpentry
+	(because Data Carpentry doesn't teach Git,
+	and therefore doesn't want to require instructors to have to know it).
+    1.  Changes to content are discouraged,
+	since the lessons are now fairly stable.
+    1.  New exercises are also becoming problematic,
+	since we already have more than we can possibly use.
+    1.  Load on lesson maintainers has become unsustainable.
+1.  **Discussion sessions.**
+    1.  The trainee then takes part in an hour-long group discussion of that lesson
+        led by an experienced instructor.
+    1.  She is expected to have gone through the lesson *before* this session
+        so that she can ask lots of pointed questions during that hour.
+    1.  If the discussion leader feels the trainee is unprepared,
+        she may be asked to do some more work and try again.
+1.  **Final demonstration lessons.**
+    1.  The trainee then does a short demonstration lesson via screen sharing.
+    1.  If the examiner feels that she needs to do more work,
+        she will be given feedback and asked to try again.
+    1.  Trainees do not have to qualify separately for different topics within one Carpentry,
+        but *do* have to do a separate demonstration lesson for each Carpentry.
```

Comment:
Should we add that the teaching material _has_ to be on one of our lessons, otherwise they are not eligible? Just to make it completely clear, that is.